pkg/cmd/generate-test-timeouts/main.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/alessio/shellescape"
+	"github.com/cockroachdb/errors"
```

Comment:
Can you use the standard library `errors` instead of `cockroachdb/errors`? `cockroachdb/errors` is a pretty heavy dependency, we don't necessarily want to force people to build it.